test/randomdog-tests.js

```diff
@@ -78,13 +78,13 @@ describe('randomdog', () => {
         })
         it('should only return jpeg dogs when include is jpg', async () => {
             // @ts-ignore
-            fsLayer.getGoodDogs.resolves(['testDog.mp4', 'testDog.JPG', 'dog.png'])
+            fsLayer.getGoodDogs.resolves(['testDog.mp4', 'testDog.jpg', 'dog.png'])
             return request(await createApp('test_host'))
                 .get('/woof.json?include=jpg')
                 .expect('Content-Type', /json/)
                 .expect(200)
                 .then(response => {
-                    expect(response.body.url).to.equal('test_host/testDog.JPG')
+                    expect(response.body.url).to.equal('test_host/testDog.jpg')

```

Comment:
decided to go with the stub for checking the file size, so this should be able to stay as uppercase.